source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -39,20 +39,117 @@ We plan to continue adding additional logs and metrics as required, add pre-defi
 **Note:** Currently it should be installed on a new machine, separate from the engine. 
 It can be installed on a dedicated VM. 
 
-Please follow one of these options. Adding SSO is optional:
- 
-  * [Metrics Store setup on top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
+Please follow the installation instructions:
+
+  * [Metrics Store setup on top of OpenShift](https://github.com/ViaQ/Main/blob/master/README-mux.md)
 
-  OR
+In oVirt 4.2 there will be an option to add SSO:
+  * [Metrics Store setup on top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
 
-  * [Metrics Store setup on top of OpenShift without oVirt Engine SSO](https://github.com/ViaQ/Main/blob/master/README-mux.md)
 
 [![oVirt Metrics data flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
 
 Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+*mux* is short for multiplex, because it acts like a multiplexor or manifold,
+taking in connections from many collectors, and distributing them to a data
+store. 
+
+1. Use SSH to connect to the logging machine and open the command line.
+
+2. Update the deployment configuration of mux pod by running the following command:
+
+       # $ oc edit dc logging-mux
+
+Edit the cpu and memory values as follows:
+
+      spec:
+        template:
+          spec:
+            containers:
+      ...
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 2Gi
+
+**Note:**
+   - CPU usage is measured in millicores (m). 500m means 0.5 cores. To allocate more resources to *mux*, increase the CPU to 1000m.
+   In the event that mux has utilized all available CPU resources, and the CPU has already been increased to 1000m, scale up to an additional pod.
+   
+   - Memory usage is measured in Gigabytes (Gi). 2Gi means 2 Gigabytes. Increase as needed.
+
+   
+3. Save the new configuration to automatically trigger a redeployment of all mux pods.
+
+4. To view when mux pod was last redeployed, run:
+
+       # oc get pods -l component=mux
+
+If mux pos was not redeployed in the last two minutes, run:
+
+       # oc rollout latest dc/logging-mux
+
+To follow the deployment until the new mux pod is rolled out, run:
+
+       # oc rollout status -w dc/logging-mux
+
+
+Since ruby isn't multi-threaded, you can also scale up mux to run additional pods:
+
+      # oc scale --replicas=2 dc/logging-mux
+
+This will create 2 mux pods.
+
+
+
+
+**Update curator pod for metrics index**
+
+This will define the curator pod to delete metrics indexes that are older then 7 days.
+
+1. Update the configmap Run:
+Run:
+       # oc edit configmap logging-curator
+
+Under 
```

Comment:
This is still part of the step since the command opened a file and now the user should edit and save it.